packages/react-components/lib/item-unknown-state.tsx

```diff
@@ -0,0 +1,27 @@
+import { makeStyles, Divider } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+
+const useStyles = makeStyles(() => ({
+  typography: {
+    padding: '0.5rem 0',
+  },
+}));
+
+interface ItemUnknownProps {
+  unknownItem: string;
+}
+
+export default function ItemUnknownState(props: ItemUnknownProps): JSX.Element {
```

Comment:
I'm wondering how useful would this be given that it is pretty much just a Typography. The only place this is used now is in the dispenser accordion and it still would need to make sure the AccordionSummary uses the unknown state.

i.e. Usage of this needs to be coupled with an appropriate AccordionSummary for it to make sense, and even so, other accordions would probably use their own unknown representation as they need to display different things.

Reply:
I wanted to use this component in the robot Accordion as well. I was thinking that for items (lift, doors, robots, dispensers etc ...) with unknown states, we can use a generic component to simply tell users that the state of the item is currently not known and there is no need for a more detailed explanation of why it is unknown. But I guess using Typography would also be just fine in the accordion itself.

Reply:
It would be nice if we have sometime more fancy than a simple typography. For now since it's very simple and not used any where else I say just have "inlined" in the dispenser accordion. You can make a separate PR to standardize all the components unknown representation if you would like to.